apps/meteor/tests/e2e/page-objects/fragments/home-content.ts

```diff
@@ -47,6 +47,10 @@ export class HomeContent {
 		return this.lastUserMessage.locator('[data-qa-type="message-body"]');
 	}
 
+	get lastUserMessageAttachment(): Locator {
+		return this.page.locator('[data-qa-type="message-attachment"]').last();
```

Comment:
Is there any way to not use `data-qa` here? Maybe try an accessible locator.

Reply:
I don't think so... since all the similar tests use the same data-qa-type to select internal <Box> components